docs/index.md

```diff
@@ -0,0 +1,199 @@
+- **command** (String) The plugin executable (absolute path, or expects the plugin to be in OS PATH).
+- **env** (Map of String) Environment values to set on the plugin process.
+
+All attributes are optional, but you must either set a config path or static credentials. An empty provider block will not be a functional configuration.
+
+Due to the internal design of this provider, access to a responsive API server is required both during PLAN and APPLY. The provider makes calls to the Kubernetes API to retrieve metadata and type information during all stages of Terraform operations.
+
+### Credentials
+
+For authentication, the provider can be configured with identity credentials sourced from either a `kubeconfig` file, explicit values in the `provider` block, or a combination of both.
+
+If the `config_path` attribute is set to the path of a `kubeconfig` file, the provider will load it and use the credential values in it. When `config_path` is not set **NO EXTERNAL KUBECONFIG WILL BE LOADED**. Specifically, $KUBECONFIG environment variable is **NOT** considered.
+
+Take note of the `current-context` configured in the file. You can override it using the `config_context` provider attribute.
+
+If both `kubeconfig` and static credentials are defined in the `provider` block, the provider will prefer any attributes specified by the static credentials and ignore the corresponding attributes in the `kubeconfig`.
+
+There are five options for providing identity information to the provider for authentication purposes:
+
+* a kubeconfig
+* a client certificate & key pair
+* a static token
+* a username & password pair
+* an authentication plugin, such as `oidc` or `exec` (see examples folder).
+
+## Experimental Status
+
+By using the software in this repository (the "Software"), you acknowledge that: (1) the Software is still in development, may change, and has not been released as a commercial product by HashiCorp and is not currently supported in any way by HashiCorp; (2) the Software is provided on an "as-is" basis, and may include bugs, errors, or other issues;  (3) the Software is NOT INTENDED FOR PRODUCTION USE, use of the Software may result in unexpected results, loss of data, or other unexpected results, and HashiCorp disclaims any and all liability resulting from use of the Software; and (4) HashiCorp reserves all rights to make all decisions about the features, functionality and commercial release (or non-release) of the Software, at any time and without any obligation or liability whatsoever.
+
+## Getting Started
+
+If this is your first time here, you can get an overview of the provider by reading our [introductory blog post](https://www.hashicorp.com/blog/deploy-any-resource-with-the-new-kubernetes-provider-for-hashicorp-terraform/).
+
+Otherwise, start by downloading a copy of our latest build from the [releases page](https://github.com/hashicorp/terraform-provider-kubernetes-alpha/releases).
```

Comment:
We don't need this once we're in the registry. In fact we should stop uploading binaries to the github release page for new builds and drive users to the registry and releases.hashicorp.com

Reply:
Good point, I'll drop it.